diff --git a/node_modules/@sentry/cli/js/helper.js b/node_modules/@sentry/cli/js/helper.js
index 56f95c9..b397122 100644
--- a/node_modules/@sentry/cli/js/helper.js
+++ b/node_modules/@sentry/cli/js/helper.js
@@ -298,6 +298,13 @@ function execute(args_1, live_1, silent_1, configFile_1) {
         if (config.customHeader) {
             env.CUSTOM_HEADER = config.customHeader;
         }
+
+        if (process.env['SENTRY_TEST_OUT_DIR']) {
+            const out = path.join(process.env['SENTRY_TEST_OUT_DIR'], 'sentry-cli-mock.json');
+            fs.appendFileSync(out, JSON.stringify(args) + ',\n');
+            return Promise.resolve();
+        }
+
         else if (config.headers) {
             const headers = Object.entries(config.headers).flatMap(([key, value]) => [
                 '--header',
